cloudpickle_generators/tests/test_cloudpickle_generators.py

```diff
@@ -124,6 +125,7 @@ def g():
     g(0)
 
 
+@pytest.mark.xfail(sys.version_info >= (3, 8, 0), reason="needs to be investigated")
```

Comment:
Whats the error here? This is an important test for the skeleton + fill pattern.

The point of this test is to make a recursive structure where one of the free variables of a generator is itself. The point of the skeleton is that it is flat, so that when we can serialize it and then fill it back up once we are done deserializing everything.